Bump to Camel 3.2.0

oscerd opened the pull request:
Fixes #199

valdar commented on connectors/camel-activemq-kafka-connector/pom.xml:
```diff
@@ -118,6 +118,18 @@
         <enabled>true</enabled>
       </releases>
     </repository>
+    <!-- camel-ipfs and camel-weka -->
```
I this repo not needed anymore with camel 3.2.0 or it wasn't ever needed?

oscerd replied:
This repo seems to be needed from 3.2.0 ahead, because of weka and camel-ipfs

oscerd commented:
> Do we know which connectors we need to remove that are not present in camel 3.2.0 anymore?

camel-eventadmin (OSGi related, moved to camel-karaf)
camel-paxlogging (OSgi related, moved to camel-karaf)
camel-spark-rest (removed in 3.2.0)
